data/remnant/remnant jobs.txt

```diff
@@ -200,7 +200,7 @@ mission "Remnant: Bounty 5"
 	job
 	repeat
 	name "Destroy Ground Assault Bounty"
-	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it, and then return to <planet> to receive your payment of <payment>."
+	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it and any fighters its carrying, and then return to <planet> to receive your payment of <payment>."
```

Comment:
```suggestion
	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it and any fighters it is carrying, and then return to <planet> to receive your payment of <payment>."
```

Reply:
I accidentally did this exact thing before realizing that you suggested that lmao